### ISSUE TITLE
do_normalize set to True by default for WAV2VEC tokenizer

### ISSUE THREAD

**Issue by Lhemamou** [huggingface/transformers#12151]
## Environment info
- `transformers` version: 4.6.1
- Platform: macOS-11.2.3-x86_64-i386-64bit
- Python version: 3.8.2
- PyTorch version (GPU?): 1.8.1 (False)
- Tensorflow version (GPU?): 2.4.1 (False)
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

### Who can help
@LysandreJik
@sgugger

## Information

Model I am using (Bert, XLNet ...): Wav2Vec

The problem arises when using:
* [*] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [ ] my own task or dataset: (give details below)

## To reproduce

wav_input_16khz, samplerate = sf.read(AUDIOFILE)
tokenizer = Wav2Vec2Tokenizer.from_pretrained("facebook/wav2vec2-base-960h")
tokenizer_2 = Wav2Vec2Tokenizer.from_pretrained("facebook/wav2vec2-base-960h",do_normalize=False)
features = tokenizer(wav_input_16khz, return_tensors="pt").input_values
features_2 = tokenizer_2(wav_input_16khz, return_tensors="pt").input_values
features == features_2

Out[1]: tensor([[False, False, False,  ..., False, False, False]])

## Expected behavior

As written in the [documentation](https://huggingface.co/transformers/_modules/transformers/models/wav2vec2/feature_extraction_wav2vec2.html#Wav2Vec2FeatureExtractor.__call__) _"do_normalize (:obj:`bool`, `optional`, defaults to :obj:`False`):
            Whether or not to zero-mean unit-variance normalize the input. Normalizing can help to significantly
            improve the performance for some models, *e.g.*, `wav2vec2-lv60
            <https://huggingface.co/models?search=lv60>`__."_  should be set to False.

However, the option seems to be set to True by default during the initialization.

**Comment by sgugger** [comment]
cc @patrickvonplaten

**Comment by patrickvonplaten** [comment]
Hey @Lhemamou, the parameter `do_normalize` is overwritten by the model's config: https://huggingface.co/facebook/wav2vec2-base-960h/blob/main/feature_extractor_config.json

**Comment by Lhemamou** [comment]
Thanks @patrickvonplaten, it solved the issue ! :) . Nonetheless, in the code from [documentation](https://huggingface.co/transformers/_modules/transformers/models/wav2vec2/feature_extraction_wav2vec2.html#Wav2Vec2FeatureExtractor.__call__), the initialization part of the class Wav2Vec2FeatureExtractor seems to initialize do_normalize to True by default, contrary to what is written in the documentation for the same class function : 

>     def __init__(
>         self,
>         feature_size=1,
>         sampling_rate=16000,
>         padding_value=0.0,
>         return_attention_mask=False,
>         do_normalize=True,
>         **kwargs
>     )
> 

and

> 
> do_normalize (:obj:`bool`, `optional`, defaults to :obj:`False`):
>             Whether or not to zero-mean unit-variance normalize the input. Normalizing can help to significantly
>             improve the performance for some models, *e.g.*, `wav2vec2-lv60

**Comment by patrickvonplaten** [comment]
Oh yeah you're right @Lhemamou ! Would you maybe like to open a PR to fix the documentation ? It should state that it defaults to `True` in this case

**Comment by Lhemamou** [comment]
sure I will do it when I have free time :)
